files/collect_api_events.rb

```diff
@@ -27,6 +28,11 @@ def main(confdir, logpath, lockdir)
   end
 
   lockfile.write_lockfile
+  if lockfile.lockfile_exists?
+    log.debug("Lockfile was successfully created:  + #{lockfile.lockfile_exists?}")
```

Comment:
Maybe just have a string instead of the interpolation? Bc we already check the existence of the lockfile in line 31. (I think we changed this in the remote session)